### PR TITLE
fix community join buttons hiding after joining

### LINKED ISSUE #14659: fix: communities join screen - join button on overview page not hidden after making a join request
# Bug Report

## Problem 
Navigate to community overview page (for both open and closed) communities. Open the bottom sheet to join that community

#### Expected behavior
clicking the join button will close the bottom sheet and return to the community overview page with the request button that opens the bottom sheet hidden

#### Actual behavior
clicking the join button will close the bottom sheet and return to the community overview page with the request button that opens the bottom sheet still showing

### PR DESCRIPTION
**Pull request by J-Son89**
fixes https://github.com/status-im/status-mobile/issues/14659

### CODE REVIEW

**Review by ilmotta — APPROVED** [review]

**Review by jo-mut — APPROVED** [review]

File: src/status_im2/contexts/communities/overview/view.cljs [hunk 1]
```diff
@@ -219,89 +219,88 @@
...
-                                                                      :amount         10
-                                                                      :is-sufficient? false
-                                                                      :is-purchasable true}
-                                                                     {:token          "RARE"
-                                                                      :token-img-src  rare-token-img
-                                                                      :amount         10
-                                                                      :is-sufficient?
-                                                                      false}]}}}])}])}]])
-        (when (or pending? joined)
+         [rn/view {:padding-horizontal 20}
```
Comment (briansztamfater) [inline src/status_im2/contexts/communities/overview/view.cljs]: Maybe we should we use `style` prop for consistency

**Review by briansztamfater — APPROVED** [review]

**Comment by churik** [comment]
@J-Son89 can you rebase it please and we'll take it to test?
Thanks!

**Comment by J-Son89** [comment]
> @J-Son89 can you rebase it please and we'll take it to test? Thanks!

thanks @churik, I am going to wait until this pr https://github.com/status-im/status-mobile/pull/14799 is merged because it simplifies the code a lot and will make my implementation up here much cleaner. :)

File: src/status_im2/common/scroll_page/view.cljs [hunk 2]
```diff
@@ -72,7 +72,7 @@
         :mid-section             {:type            :text-with-description
                                   :main-text       nil
                                   :description-img nil}
-        :right-section-buttons   (:right-section-buttons page-nav)
+        :right-section-buttons    page-nav
```
Comment (J-Son89) [inline src/status_im2/common/scroll_page/view.cljs]: fixed as it was not showing before

**Comment by J-Son89** [comment]
@churik this pr is ready for test now :)

**Comment by VolodLytvynenko** [comment]
@J-Son89 sorry for the delay in testing. Just took this PR to work. The feedback will be provided soon. 
Could you please rebase it?

**Comment by J-Son89** [comment]
@VladimrLitvinenko, np! I just rebased and pushed now :)

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 thank you for PR. Here is the one detected issue:

**ISSUE 1: The request is not sent after the 'request to join' button is tapped on the closed community**

**Preconditions:**
User_A and User_B are mutual contacts

**Steps to reproduce:**
1. Create a closed community as User_A
2. Share current community to User_B
3. Open the discovery page as User_B
4. Tap on the shared community -> tap 'Request to join' button

**Actual result:**
The community detailed page is still shown with the enabled 'Request to join' button and current request is not received by the admin of the community

https://user-images.githubusercontent.com/52490791/215537725-7c52ac5e-49c3-453e-8683-248c1571159a.mp4


**Expected result:**
The admin received the request

**Comment by J-Son89** [comment]
@VladimrLitvinenko, I fixed that issue. Thanks for finding it! :)

**Comment by VolodLytvynenko** [comment]
hi @J-Son89 Looks like some integration tests are not passed. Could you check it please? https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-14761/10/consoleText

**Comment by J-Son89** [comment]
> hi @J-Son89 Looks like some integration tests are not passed. Could you check it please? https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-14761/10/consoleText

apologies @VladimrLitvinenko, sorted now! :D

**Comment by VolodLytvynenko** [comment]
hi @J-Son89. The issue is fixed. Thank you for your work. No issues from my side. Could you merge this PR?

**Comment by J-Son89** [comment]
> hi @J-Son89. The issue is fixed. Thank you for your work. No issues from my side. Could you merge this PR?

thanks! Will merge it now :)

**Pull request merged**